tests/unit/s2n_resume_test.c

```diff
@@ -120,15 +120,8 @@ int main(int argc, char **argv)
             /* Result matches actual size of data */
             DEFER_CLEANUP(struct s2n_stuffer actual_data = { 0 }, s2n_stuffer_free);
             EXPECT_SUCCESS(s2n_stuffer_growable_alloc(&actual_data, actual_size));
-            EXPECT_OK(s2n_tls13_serialize_resumption_state(conn, &actual_data));
-            const uint32_t expected_size = s2n_stuffer_data_available(&actual_data);
-            if (actual_size != expected_size) {
-                fprintf(stderr, "\nS2N_TLS13_FIXED_STATE_SIZE (%i) should be set to %u\n\n",
-                        S2N_TLS13_FIXED_STATE_SIZE, expected_size);
-            }
-            EXPECT_EQUAL(actual_size, expected_size);
-
-            EXPECT_SUCCESS(s2n_connection_free(conn));
+            /* Session tickets can't have zero-length secrets */
+            EXPECT_ERROR(s2n_tls13_serialize_resumption_state(conn, &actual_data));
```

Comment:
Please add an explicit test for "Session tickets can't have zero-length secrets". The test cases for s2n_connection_get_session_state_size are not the right place for that test.